### DISCUSSION TITLE
logging.max_history_files

### DISCUSSION THREAD

**Discussion by houzhx759** [apache/skywalking#7796] [Q&A]
How should I close it by mounting logging.max_history_files =-1 in the env environment variable? I wrote in the YAML file - name: SW_LOGGING_MAX_HISTORY_FILES
Value: 0 Jenkins failed to package it. My agent is connected to Skywalking through sidecar mode. Please help me.
![image](https://user-images.githubusercontent.com/86034885/134653159-48a3ce36-228f-4242-9daa-113c60b76fac.png)

**Comment by wu-sheng** [comment]
Read document, could you change log level to OFF. If you want to skip #7792. You will face more serious issue, even crash later. Don't try to ignore all issues.

**Reply by houzhx759** [reply]
I read the official documentation to turn off the rolled-log set to -1 or 0, but I can't pass in the environment variable, my YAML file is not deployable
![image](https://user-images.githubusercontent.com/86034885/134656214-050e63f1-2708-4e27-91fa-4e93961b159b.png)

**Reply by wu-sheng** [reply]
`level` and `output` are there for you to control this kind of things.

**Reply by houzhx759** [reply]
Should I change logging.level=${SW_LOGGING_LEVEL:INFO} to logging.level=${SW_LOGGING_LEVEL:DEBUG} so that flipped logs can be turned off?
